specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -2876,11 +2876,6 @@
           "$ref": "#/definitions/PrivateEndpointConnectionProperties",
           "x-ms-client-flatten": true,
           "description": "Resource properties."
-        },
-        "type": {
-          "readOnly": true,
```

Comment:
please add id, type and name explicitly and remove reference to proxyOnlyResource

Reply:
added